lib/ecmascript.ts

```diff
@@ -1364,8 +1394,9 @@ export function ToTemporalTimeRecord(
   bag: Partial<Record<keyof TimeRecord, string | number | undefined>>,
   completeness: FieldCompleteness = 'complete'
 ): Partial<TimeRecord> {
-  // NOTE: Field order here is important.
+  // NOTE: Field order is sorted to make the sort in PrepareTemporalFields more efficient.
```

Comment:
Couldn't we just remove this comment entirely? Presumably the field order here doesn't matter at all now as it will always be sorted?

Reply:
I didn't know there was a fast-path. Happy to keep the comment as-is.